openpaas-james/deployment-tests/distributed-ldap/src/test/java/deployment/OpenpaasJamesDistributedLdapExtension.java

```diff
@@ -52,10 +55,11 @@ public OpenpaasJamesDistributedLdapExtension() {
 
     @SuppressWarnings("resource")
     private GenericContainer<?> createOpenPaasJamesDistributedLdap() {
-        return new GenericContainer<>("linagora/openpaas-james-distributed-ldap:latest")
+        return new GenericContainer<>("linagora/openpaas-james-distributed:latest")
             .withNetworkAliases("james-distributed-ldap")
             .withNetwork(network)
             .dependsOn(cassandra, elasticsearch, s3, rabbitmq, ldap)
+            .withCopyFileToContainer(MountableFile.forClasspathResource("james-conf/usersrepository.xml"), "/root/conf/")
```

Comment:
:+1: glad it works on the CI!

Reply:
Yep this one works, compared to the other one :)))